tests/server/server-executeQuery-007.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\Server::executeQuery() with negative limit returns a single batch
+MongoDB\Driver\Server::executeQuery() with limit and single batch
```

Comment:
Decided to repurpose this test instead of delete it, as we didn't have any other PHPC tests for `singleBatch: true`.